### PR TITLE
Docs improvement

### PR DESCRIPTION
**Pull request by federicobozzini**
Some minor docs improvements.

### CODE REVIEW

File: CLI_QUICKSTART.markdown [hunk 1]
```diff
@@ -201,7 +201,7 @@ $ vc-pgen \
     --capability "/program/:x,/output/:r" \
     --binary /program/example-binary.wasm=example/example-binary.wasm \
     --capability "/input/:r,/output/:w" \
-    --output-policy-file example/example-policy.json
```
Comment (federicobozzini) [inline CLI_QUICKSTART.markdown:204]: I think there is a missing character here.

File: README.markdown [hunk 2]
```diff
@@ -26,7 +26,6 @@ The latest project news is available on the [Veracruz project homepage](https://
 To jump straight into the codebase, please read the growing number of guides we have on how to get started with Veracruz:
 - [BUILD_INSTRUCTIONS.markdown](BUILD_INSTRUCTIONS.markdown) - Set up a build environment for Veracruz
 - [CLI_QUICKSTART.markdown](CLI_QUICKSTART.markdown) - Quickly run Veracruz with a prepared demo program
-- [CLI_INSTRUCTIONS.markdown](CLI_INSTRUCTIONS.markdown) - A more in-depth guide to running Veracruz, including steps on how to generate certificates and the policy file.
```
Comment (federicobozzini) [inline README.markdown:29]: This link is dead. I think only `CLI-INSTRUCTIONS.markdown is used now according to [this](
https://github.com/veracruz-project/veracruz/commit/bc9802c2e5ca496485a365b5f18e9c4af0a7eab4).

**Comment by dominic-mulligan-arm** [comment]
🎉 Thanks @federicobozzini, and welcome to the Veracruz project! 🎉

Can you amend this PR to also add your name and email address to the `AUTHORS.markdown` file?

**Comment by federicobozzini** [comment]
> Can you amend this PR to also add your name and email address to the `AUTHORS.markdown` file?

Done, thank you very much!

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**